dockers/docker-database/docker-database-init.sh

```diff
@@ -30,15 +30,20 @@ fi
 
 if [[ $IS_DPU_DEVICE == "true" ]]
 then
-    midplane_ip=$( ip -4 -o addr show eth0-midplane | awk '{print $4}' | cut -d'/' -f1  )
-    if [[ $midplane_ip != "" ]]
-    then
+    if [[ $IS_APPLIANCE_DPU == "false" ]]; then
```

Comment:
It is better to change to `if [[ $IS_APPLIANCE_DPU != "true" ]]; then`. In this case the code will work correctly even if IS_APPLIANCE_DPU is not set